modules/apps/object/object-rest-test/src/testIntegration/resources/com/liferay/object/rest/internal/resource/v1_0/test/dependencies/expected_openapi.json

```diff
@@ -494,6 +494,11 @@
 						"type": "string"
 					}
 				},
+				"required": [
```

Comment:
@albertojml shouldn't this be included separately in each of the `oneOf`s? If not they will both the 3 of them be required.